templates/idc/quota.html

```diff
@@ -19,47 +18,13 @@
 
 {% endcomment %}
 
-{% block header %}
-
-{% endblock %}
-{% block link_page_name %}landing{% endblock %}
-{% block page_name %}landing{% endblock %}
 {% block content %}
-
-    <div class="landing-content-div">
-        <div class="lj-overlay lj-overlay-bg"></div>
-        <div class="lj-overlay lj-overlay-color lj-overlay-image"></div>
-        <div class="container">
-            <div class="header-div">
-                <div class="header-content-div">
-                    <div class="row">
-                        <div class="col-xs-12 lj-title">
-                            <h1 class="lj-text-center"><span>Daily Imaging Data Quota Exceeded</span></h1>
-                        </div>
-                    </div>
-                    jkl {{ request.session.name }} jkl
-                    <div class="col-xs-8 col-xs-offset-2 lj-title-paragraph">
-                        <p class="lj-text-center">Users are restricted to a quota of {{ quota }} MB per day of medical imaging files. You have reached your daily quota. Your quota will be reset at midnight GMT.</p>
-                    </div>
-
-
-
-
-
-
-
-
-
-                </div>
+    <div>
+        <div class="quota-body">
+            <div class="center">
+                <h1 class="center"><span>Daily Imaging Data Quota Exceeded</span></h1>
+                <p>Users are restricted to a quota of {{ quota }} GB per day of medical imaging files. You have reached your daily quota. Your quota will be reset at midnight GMT.</p>
             </div>
-
-
-
         </div>
     </div>
-{% endblock %}
-
-{% block js_file %}
```

Comment:
So we have to leave this block in, or bootstrap won't load and everything breaks. If we need to disable something (eg. the warning...which I'm working on) we need to do that separately.

Reply:
Make sense!  Added back. Thanks!